app/modules/github/github_service.py

```diff
@@ -254,7 +254,36 @@ async def get_repos_for_user(self, user_id: str):
             raise HTTPException(
                 status_code=500, detail=f"Failed to fetch repositories: {str(e)}"
             )
-
+    async def get_combined_user_repos(self, user_id: str):
+        parsed_repos = await self.project_manager.list_projects(user_id)
+        project_list = [
+            {
+                "id": project["id"],
+                "name": project["repo_name"].split('/')[-1],
+                "full_name": project["repo_name"],
+                "private": False,
+                "url": f"https://github.com/{project['repo_name']}",
+                "owner": project["repo_name"].split('/')[0]
+            }
+            for project in parsed_repos
+        ]
+        user_repo_response = await self.get_repos_for_user(user_id)
```

Comment:
filter all the github response repos from unique list from db, then mark all the db projects as public